prac_06/guitar.py

```diff
@@ -0,0 +1,24 @@
+"""Guitar class"""
+
+
+class Guitar:
+    """Class for a guitar object"""
+
+    def __init__(self, name="", year=0, cost=0):
+        """Initialise the guitar object."""
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        """Format the guitar object in a string format."""
+        return f"{self.name} ({self.year}) : ${self.cost:,.2f}"
+
+    def get_age(self):
+        """Calculate the age of the guitar from the current year."""
+        age = 2023 - self.year
```

Comment:
Could have maybe used a constant variable for the CURRENT_YEAR but otherwise, looks good